### PR TITLE
Remove HTTP Redirect

### PR DESCRIPTION
**Pull request by OceanOak**
Changelog:

```
Area of change
- Remove Http Redirect
```

### CODE REVIEW

File: backend/src/LibExecution/DvalReprInternalRoundtrippable.fs [hunk 1]
```diff
@@ -43,7 +41,7 @@ module FormatV0 =
     | DObj of DvalMap
     | DError of DvalSource * string
     | DIncomplete of DvalSource
-    | DHttpResponse of DHTTP
+    | DHttpResponse of (int64 * List<string * string> * Dval)
```
Comment (pbiggar) [inline backend/src/LibExecution/DvalReprInternalRoundtrippable.fs:44]: FYI: including the parens here means something different, so you can remove them.

**Comment by OceanOak** [comment]
For the tests that needs to be re-enabled:
when using `DStr("")` in the redirectTo function they pass but when changing it to `DBytes([||])` and updating the tests accordingly, it fails.

Tests:
```
(Http.redirectTo_v0 "http://someothersite.com") = Http.responseWithHeaders [||] { Location = "http://someothersite.com" } 302
(Http.redirectTo_v0 "/relativeUrl") = Http.responseWithHeaders [||] { Location = "/relativeUrl" } 302
```
Error:
```
expected: DList []
  actual: [||]",
```
Is there a different way to send an empty byte array?

**Comment by StachuDotNet** [comment]
`Bytes::empty` exists

**Comment by OceanOak** [comment]
> Try `Bytes::empty`

I get Unsupported expression

**Comment by StachuDotNet** [comment]
Sorry, try Bytes.empty - forgot `::` doesn't work in test files.

**Comment by OceanOak** [comment]
> Sorry, try Bytes.empty - forgot `::` doesn't work in test files.

This worked. Thanks!

**Pull request merged**